cmd/kube-scheduler/app/server.go

```diff
@@ -214,7 +214,7 @@ func (o *Options) applyDeprecatedHealthzPortToConfig() {
 // 3. --algorithm-provider to use a named algorithm provider.
 func (o *Options) applyDeprecatedAlgorithmSourceOptionsToConfig() {
 	switch {
-	case o.useLegacyPolicyConfig:
+	case o.useLegacyPolicyConfig || (len(o.policyConfigFile) > 0 && o.policyConfigMapName == ""):
```

Comment:
s/o.policyConfigMapName == ""/len(o.policyConfigMapName) == 0/

Reply:
@bsalamat - Shall I create another PR for this?

Reply:
@ravisantoshgudimetla No, thers is no need to send another PR.
Thanks for checking.

Reply:
Ops :)